There is a missing ) in the #else branch which is not unused on Linux.

--- a/src/threads.h
+++ b/src/threads.h
@@ -303,7 +303,7 @@ })
 #else
 #define SCSetThreadName(n) ({ \
     strlcpy(t_thread_name, n, sizeof(t_thread_name)); \
-}
+})
 #endif
 
 
